README.md

```diff
@@ -145,7 +141,7 @@ Available targets:
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 5.20.0 |
```

Comment:
I don't know why it's doing this to you locally... but let's deal with it the dumb way: 
```suggestion
| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.0 |
```